src/pages/console/uibuilder/collections.mdx

```diff
@@ -20,18 +37,49 @@ Amplify Studio offers a visual way to bind collections to items in your data mod
 
 ### Set filters
 
-Filters control what records are displayed in a collection.
+Filters control what records are displayed in a collection. You can add multiple filters to a single collection.
```

Comment:
Did we test this?

Reply:
Good catch - no, we didn't, and it doesn't work. Confirming expected behavior with the team.

Reply:
Albert has fixed this in the newest versions!